comms/src/builder/builder.rs

```diff
@@ -284,17 +291,22 @@ where
         node_identity: Arc<NodeIdentity>,
         connection_manager_requester: ConnectionManagerRequester,
         shutdown_signal: ShutdownSignal,
-    ) -> OutboundMessageService<TOutStream, BoxedBackoff>
+    ) -> (OutboundMessageService<TOutStream, BoxedBackoff>, OutboundEventPublisher)
     {
         let outbound_stream = self.outbound_stream.take().expect("outbound_stream cannot be None");
         let oms_backoff = self.oms_backoff.take().expect("oms_backoff was None");
-        OutboundMessageService::with_backoff(
-            self.outbound_service_config.take().unwrap_or_default(),
-            outbound_stream,
-            node_identity,
-            connection_manager_requester,
-            shutdown_signal,
-            oms_backoff,
+        let (event_tx, _) = broadcast::channel(10);
```

Comment:
This buffer size seems a little low for an event bus? I can't remember if this creates back pressure or it throws away the older messages